native/package.json

```diff
@@ -63,28 +63,28 @@
     "query-string": "^6.11.0",
     "react": "16.13.1",
     "react-i18next": "^11.8.8",
-    "react-native": "0.63.3",
-    "react-native-elements": "^2.0.3",
-    "react-native-fast-image": "^7.0.2",
-    "react-native-gesture-handler": "^1.9.0",
+    "react-native": "0.63.4",
+    "react-native-elements": "^3.3.0",
+    "react-native-fast-image": "^8.3.4",
+    "react-native-gesture-handler": "^1.10.3",
     "react-native-highlight-words": "^1.0.1",
     "react-native-image-zoom-viewer": "^3.0.1",
     "react-native-inappbrowser-reborn": "^3.5.1",
-    "react-native-localize": "^2.0.1",
-    "react-native-pdf": "6.2.0",
-    "react-native-permissions": "2.0.9",
+    "react-native-localize": "^2.0.2",
+    "react-native-pdf": "6.3.0",
+    "react-native-permissions": "3.0.1",
     "react-native-progress": "^4.0.3",
-    "react-native-reanimated": "^1.13.2",
+    "react-native-reanimated": "^2.0.0",
     "react-native-render-html": "^5.1.0",
-    "react-native-safe-area-context": "^3.1.9",
+    "react-native-safe-area-context": "^3.2.0",
     "react-native-safe-area-view": "2.0.0",
-    "react-native-screens": "^2.16.1",
+    "react-native-screens": "^2.18.1",
     "react-native-static-server": "^0.5.0",
     "react-native-svg": "^12.1.0",
-    "react-native-vector-icons": "^7.1.0",
-    "react-native-webview": "10.8.2",
+    "react-native-vector-icons": "^8.1.0",
+    "react-native-webview": "11.2.3",
```

Comment:
Must not be upgraded as long as this issue is present: https://github.com/react-native-webview/react-native-webview/issues/1741

Reply:
done, reverted